content/applications/essentials/search.rst

```diff
@@ -183,6 +183,68 @@ It is possible to *customize groups* by using a field present on the model. To d
+   For some reports, the :guilabel:`Comparison` section only appears in the :guilabel:`Search...`
+   bar drop-down menu if one or more time periods is selected in the :guilabel:`Filters` column.
+   This is because, if no time period is specified, there is nothing to make a comparison to.
+
+   Additionally, some reports only allow use of the :guilabel:`Comparison` feature when the
+   :icon:`fa-pie-chart` :guilabel:`(pie chart)` graph type or the :icon:`oi-view-pivot`
+   :guilabel:`(pivot)` view is selected. A :guilabel:`Comparison` option can be selected even if
+   another view is enabled, but doing so does not change the way data is displayed on the report.
+
+.. image:: search/comparison-section.png
+   :align: center
+   :alt: The Search... bar for the production analysis report.
+
+To view data using one of the two comparisons, begin by selecting a time period in the
+:guilabel:`Filters` column of the :guilabel:`Search...` bar drop-down menu. Then, select either
+:guilabel:`(Time Filter): Previous Period` or :guilabel:`(Time Filter): Previous Year` in the
+:guilabel:`Comparison` section.
+
+With one of the :guilabel:`Comparison` options enabled, the report compares the data for the
+selected period, with the data for the same unit of time (month, quarter, year), one period or year
+prior. The way the data is displayed depends on the selected view:
+
+- The bar chart shows two bars side-by-side for each time unit of the selected time period. The left
+  bar represents the selected time period, while the right bar represents the previous time period.
+- The line chart is displayed with two lines, one representing the selected time period, and the
+  other representing the previous time period.
+- The pie chart appears as a large circle with a smaller circle inside of it. The larger circle
+  represents the selected time period, while the smaller circle represents the previous time period.
+- The pivot table is displayed with each column split into two smaller columns. The right column
+  represents the selected time period, while the left column represents the previous time period.
+
+.. example::
+   In the :guilabel:`Production Analysis` report of the :menuselection:`Manufacturing` app,
+   :guilabel:`Q2` is selected in the :guilabel:`End Date` filter section of the
+   :guilabel:`Search...` bar drop-down menu. In the :guilabel:`Comparison` section, :guilabel:`End
+   Date: Previous Year` is selected.
+
+   The current year is 2024, so the larger circle shows data for the second quarter (Q2) of 2024.
+   The smaller circle shows data for the second quarter of 2023, which is the same time period, but
+   one *year* prior.
+
+   If :guilabel:`End Date: Previous Period` is selected instead, the smaller circle shows data for
+   the first quarter of 2024, which is the same time period, but one *period* prior.
+
+   .. image:: search/comparison.png
+      :align: center
+      :alt: The comparison view of the Production Analysis report.
```

Comment:
it might be beneficial to recapture this screenshot, to show the configurations that are made in the 'Search...' bar, to make it simple for the reader to understand/recreate on their own, should they find themselves confused.